metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -1199,6 +1199,126 @@ def create_tag(self, tag_name: str) -> str:
         # return urn
         return res["createTag"]
 
+    def _assertion_result_shared(self) -> str:
+        fragment: str = """
+             fragment assertionResult on AssertionResult {
+                 type
+                 rowCount
+                 missingCount
+                 unexpectedCount
+                 actualAggValue
+                 externalUrl
+                 nativeResults {
+                     value
+                 }
+                 error {
+                     type
+                     properties {
+                         value
+                     }
+                 }
+             }
+        """
+        return fragment
+
+
+    def _run_assertion_result_shared(self) -> str:
+        fragment: str = """
+            fragment runAssertionResult on RunAssertionResult {
+                assertion {
+                    urn
+                }
+                result {
+                    ... assertionResult
+                }
+            }
+        """
+        return fragment
+
+
+    def run_assertion(self, urn: str, save_result: bool = False):
```

Comment:
Let's actually make the result saved by default! I think this will be the user expectations

Reply:
Done